### PR TITLE
Polish translation update (RAPPS and fixes)

### PR DESCRIPTION
**Pull request by archeYR**
Hello, this update translates descriptions of some applications in RAPPS and fixes/improves my translations from the last update.

### CODE REVIEW

**Comment by ThFabba** [comment]
If you don't mind, I'm going to squash all these commits into one and push the result to your branch. This will avoid merge commits (which we don't allow) as well as conflicts.

**Comment by archeYR** [comment]
Sure.

**Comment by archeYR** [comment]
Ok, everything is correct now for sure, sorry that I didn't noticed it before. Ready for merge.

File: media/rapps/ghostview.txt [hunk 1]
```diff
@@ -17,6 +17,9 @@ Size = 1,9 MiB
 [Section.0a]
 Description = Una interfaz gráfica para Ghostscript.
 
+[Section.0415]
+Description = Interfejs graficzny dla Ghostscript
```
Comment (Saibamen) [inline media/rapps/ghostview.txt]: Missing dot

**Pull request merged**